test/model/test_deepvar.py

```diff
@@ -102,9 +102,9 @@ def test_deepvar(
         freq=metadata.freq,
         distr_output=distr_output,
         scaling=False,
-        use_copula=use_copula,
+        use_marginal_transformation=use_copula,
```

Comment:
you should probably update use_copula here for consistency.

Reply:
Missed to update that file. Fixed. Thank you!